knpu/ep1-intro/routing-wildcards.md

```diff
@@ -19,6 +19,10 @@ To test this is, change the message in the response to `'The genus: '.$genusName
 
 [[[ code('086c4c19c1') ]]]
 
+***TIP
+Returning a raw response should only be done in development as it will introduce a security issue called XSS. You can [read more about XSS here.](https://www.owasp.org/index.php/Cross-site_Scripting_%28XSS%29)
```

Comment:
I prefer to use link to the wikipedia here:
https://en.wikipedia.org/wiki/Cross-site_scripting